From 35b51f7331775190b2ce10863c8065897c72870b Mon Sep 17 00:00:00 2001
Message-ID: <35b51f7331775190b2ce10863c8065897c72870b.1731259934.git.mjg@fedoraproject.org>
From: Michael J Gruber <mjg@fedoraproject.org>
Date: Sun, 10 Nov 2024 18:31:08 +0100
Subject: [PATCH] unbreak local builds

Do not determine mupdf version from a local tree if we do not have one.
We do not build mupdf in this case anyways.
---
 setup.py | 4 +++-
 1 file changed, 3 insertions(+), 1 deletion(-)

diff --git a/setup.py b/setup.py
index 79b8a2a9..ae474de0 100755
--- a/setup.py
+++ b/setup.py
@@ -553,7 +553,9 @@ def build():
     # Download MuPDF.
     #
     mupdf_local, mupdf_location = get_mupdf()
-    mupdf_version_tuple = get_mupdf_version(mupdf_local)
+    if mupdf_local:
+        mupdf_version_tuple = get_mupdf_version(mupdf_local)
+    # else we cannot determine version this way and do not use it
 
     build_type = os.environ.get( 'PYMUPDF_SETUP_MUPDF_BUILD_TYPE', 'release')
     assert build_type in ('debug', 'memento', 'release'), \
-- 
2.47.0.346.g8c6d622db9

